packages/node/src/lib/token-manager-noop.ts

```diff
@@ -0,0 +1,17 @@
+import type { AccessToken, OAuthSettings, TokenManager } from './types'
+
+export class TokenManagerNoOp implements TokenManager {
```

Comment:
Nit/no blocking: I think a fairly idiomatic convention is prefixing this kind of class like "NullTokenManager". We already use "NullStats", so might be good to follow that pattern.

Reply:
Changed.  NoOp is more clear, but it does seem to be an idiom and we already use it :}